src/core/mod.rs

```diff
@@ -50,9 +50,21 @@ pub const INITIAL_MINING_BONUS_WINDOW: u16 = 10_000;
 // first burnchain block hash
 // TODO: update once we know the true first burnchain block
 pub const FIRST_BURNCHAIN_CONSENSUS_HASH: ConsensusHash = ConsensusHash([0u8; 20]);
-pub const FIRST_BURNCHAIN_BLOCK_HASH: BurnchainHeaderHash = BurnchainHeaderHash([0u8; 32]);
-pub const FIRST_BURNCHAIN_BLOCK_HEIGHT: u32 = 0;
-pub const FIRST_BURNCHAIN_BLOCK_TIMESTAMP: u64 = 0;
+
+pub const BITCOIN_MAINNET_FIRST_BLOCK_HEIGHT: u64 = 661389;
```

Comment:
This is still really TBD, right?  We won't know what this value needs to be until the `.miners` namespace is fully populated.

Reply:
Yep -- this should have a greppable comment like

```
// TODO: TO BE SET BY STACKS_V1_MINER_THRESHOLD
```

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2173/commits/dbe8f0a60efc8d6cb6844d466422018847724ef3